zuul.d/edpm_multinode.yaml

```diff
@@ -0,0 +1,169 @@
+---
+# Base job definition for multinode. Provide base layout with CRC on a dedicated
+# nodeset, one compute and an ansible-controller.
+# At the end, you get an env ready to run the first stages of the ci-framework.
+# It also ensures post-run are common to all of the multinodes jobs we'll get
+# in that file.
+# In case you change the "nodeset" for your own job, please override
+# crc_ci_bootstrap_networking using *extra-vars*.
+- job:
+    name: cifmw-podified-multinode-edpm-base-crc
+    parent: base-extracted-crc
+    timeout: 10800
+    attempts: 1
```

Comment:
@pablintino in case Vexx is playing nasty with its DB, we may want to increase that value.